src/twisted/mail/relaymanager.py

```diff
@@ -365,7 +365,7 @@ def hasWaiting(self):
         @return: C{True} if messages are waiting to be relayed.  C{False}
             otherwise.
         """
-        return len(self.waiting) > 0
+        return bool(len(self.waiting))
```

Comment:
Out of curiosity, what is the motivation for this change? If the intent is to avoid extra operations, would `bool(self.waiting)` be better?